### PR TITLE
Landing page added

### PR DESCRIPTION
**Pull request by ibramoha**

### CODE REVIEW

File: frontend/src/Components/LandingPage/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
```
Comment (kkarimi) [inline frontend/src/Components/LandingPage/index.js]: Why is there a line between each line of your code? Can you please configure your editor to use Prettier? maybe read this: https://scottsauber.com/2017/06/10/prettier-format-on-save-never-worry-about-formatting-javascript-again/ and/or talk to Mohammad or me

File: frontend/src/Components/LandingPage/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          <div className="google-logo">
+
+            <img src="/google-logo.png" alt="google-logo" />
+
+          </div>
+
+          <div className="google-title">
+
+            <h5>Sign in with google</h5>
```
Comment (kkarimi) [inline frontend/src/Components/LandingPage/index.js]: Please use [react-router link](https://reacttraining.com/react-router/web/api/Link) to make this go to the next page (e.g.  `/services/healthcare`)

File: frontend/src/Components/LandingPage/index.js (hunk 1 above)
Comment (ibramoha) [reply frontend/src/Components/LandingPage/index.js]: Thanks. OK I will soon.

File: frontend/src/Components/LandingPage/index.js (hunk 2 above)
Comment (ibramoha) [reply frontend/src/Components/LandingPage/index.js]: Cool.

File: frontend/src/Components/LandingPage/landing-page.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
+.landing-page{
+
```
Comment (kkarimi) [inline frontend/src/Components/LandingPage/landing-page.css]: In the CSS too?

Comment (ibramoha) [reply frontend/src/Components/LandingPage/landing-page.css]: ok.

Comment (kkarimi) [reply frontend/src/Components/LandingPage/landing-page.css]: CSS file is still the same, every second line an empty line

**Pull request merged**